suites/reef/rbd/tier-2_rbd_namespace.yaml

```diff
@@ -83,3 +83,45 @@ tests:
         rep-pool-only: True
         rep_pool_config:
           rbd: {}
+
```

Comment:
same applies to this suite as well.

Reply:
not sure what are you trying to say, like I explained above, the current automation for ns is for rep-pool only, in future we will support ec-pool also.